### DISCUSSION TITLE
Installation and usage

### DISCUSSION THREAD

**Discussion by theevl** [tjltjl/plycutter#15] [Q&A] [answered]
This is a great looking project.
I think I have performed the install as instructed.

I have completed the pytest, and I think all steps succeeded. It tested 48 items and all 48 passed.

I am now trying to run the demo sample but I am getting a command not found:

(base) theevl@theevl-desktop:~/Desktop/plycutter-master$ plycutter -o foo.dxf --thickness 6 ./tests/data/PlyTest0.1.stl
plycutter: command not found
(base) theevl@theevl-desktop:~/Desktop/plycutter-master$ 

I cannot see a file called "plycutter.py", so I do not know what do to next.
Thanks for any guidance you can provide.
thanks

**Comment by traverseda** [comment, marked as answer]
You need to run `python setup.py install` to install the cli program into your virtualenv. I think that step was missing from the install guide.

I recommend installing it using pipx, [once you have pipx installed](https://github.com/pipxproject/pipx) you can easily install plycutter into it's own virtualenv using `pipx install git+https://github.com/tjltjl/plycutter.git`.

**Reply by theevl** [reply]
Thanks a lot 
I installed pipx and then installed plycutter (with pipx), and it worked right away.
this is great.

Kippos!

Thanks!


![Screenshot from 2020-12-13 22-40-58](https://user-images.githubusercontent.com/12778936/102038075-55456f00-3d94-11eb-85ad-65386e9cb9b8.png)

**Comment by tjltjl** [comment]
Today ran into the same omission from the docs, fixing now
